decred_wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -37,6 +37,7 @@ class TransactionHistoryViewController: UIViewController {
         initFilterBtn()
         self.tableView.addSubview(self.refreshControl)
         self.tableView.register(UINib(nibName: TransactionTableViewCell.identifier, bundle: nil), forCellReuseIdentifier: TransactionTableViewCell.identifier)
+        self.tableView.tableFooterView = UIView.init()
```

Comment:
Why is this necessary?

Reply:
This removes empty lines on tableview when there is no data on the list